src/pipelines.js

```diff
@@ -232,27 +302,51 @@ class Text2TextGenerationPipeline extends Pipeline {
 
         let outputTokenIds = (await this.model.generate(input_ids, generate_kwargs)).flat();
 
+        /**
+         * @type {any[]}
+         */
         let toReturn = this.tokenizer.batch_decode(outputTokenIds, {
             skip_special_tokens: true,
         });
         if (this._key !== null) {
-            toReturn = toReturn.map(text => {
+            toReturn = toReturn.map((text) => {
```

Comment:
(simply back to original)

```suggestion
            toReturn = toReturn.map(text => {
```